src/components/Selector.tsx

```diff
@@ -52,6 +53,7 @@ export function Selector<
   children,
   classes,
   multiple,
+  allSelectable = false as any,
```

Comment:
nit: no need for default value assignment, will be `undefined` by default which will make the "Select All" action hidden.